frontend/server/src/Controllers/TeamsGroup.php

```diff
@@ -335,6 +339,147 @@ public static function apiRemoveTeam(\OmegaUp\Request $r): array {
+                $alias
+            )
+        );
+        $team = \OmegaUp\DAO\TeamGroups::getByTeamUsername($teamUsername);
+        if (is_null($team)) {
+            throw new \OmegaUp\Exceptions\InvalidParameterException(
+                'parameterNotFound',
+                'teams_group_alias'
+            );
+        }
+
+        $teamsGroup = self::validateTeamGroupAndOwner(
+            $team['alias'],
+            $r->identity
+        );
+        if (is_null($teamsGroup) || is_null($teamsGroup->team_group_id)) {
+            throw new \OmegaUp\Exceptions\InvalidParameterException(
+                'parameterNotFound',
+                'teams_group_alias'
+            );
+        }
+
+        $identitiesUsernames = explode(',', $r->ensureString('usernames'));
+
+        try {
+            \OmegaUp\DAO\TeamUsers::createTeamUsersBulk(
+                $team['team_id'],
+                $identitiesUsernames
+            );
+        } catch (\Exception $e) {
+            throw $e;
+        }
+
+        return ['status' => 'ok'];
+    }
+
+    /**
+     * Remove an existing team member of a teams group
+     *
+     * @return array{status: string}
+     *
+     * @omegaup-request-param string $team_group_alias
+     * @omegaup-request-param string $username
+     */
+    public static function apiRemoveMember(\OmegaUp\Request $r) {
+        $r->ensureMainUserIdentity();
+
+        $teamUsername = $r->ensureString(
+            'team_group_alias',
+            fn (string $alias) => \OmegaUp\Validators::usernameOrTeamUsernameOrEmail(
+                $alias
+            )
+        );
+        $team = \OmegaUp\DAO\TeamGroups::getByTeamUsername($teamUsername);
+        if (is_null($team)) {
+            throw new \OmegaUp\Exceptions\InvalidParameterException(
+                'parameterNotFound',
+                'teams_group_alias'
+            );
+        }
+
+        $teamsGroup = self::validateTeamGroupAndOwner(
+            $team['alias'],
+            $r->identity
+        );
+        if (is_null($teamsGroup) || is_null($teamsGroup->team_group_id)) {
+            throw new \OmegaUp\Exceptions\InvalidParameterException(
+                'parameterNotFound',
+                'teams_group_alias'
+            );
+        }
+
+        $user = \OmegaUp\DAO\Users::FindByUsername(
+            $r->ensureString(
+                'username'
+            )
+        );
+        if (is_null($user) || is_null($user->user_id)) {
+            throw new \OmegaUp\Exceptions\InvalidParameterException(
+                'parameterNotFound',
+                'user_id'
+            );
+        }
+
+        try {
+            \OmegaUp\DAO\TeamUsers::delete(
+                new \OmegaUp\DAO\VO\TeamUsers([
+                    'team_id' => $team['team_id'],
+                    'user_id' => $user->user_id,
+                ])
+            );
+        } catch (\Exception $e) {
+            throw $e;
+        }
```

Comment:
```suggestion
        \OmegaUp\DAO\TeamUsers::delete(
            new \OmegaUp\DAO\VO\TeamUsers([
                'team_id' => $team['team_id'],
                'user_id' => $user->user_id,
            ])
        );
```

no tiene mucho sentido atrapar algo para arrojarlo inmediatamente sin hacer nada al respecto.

Reply:
Enterado. Ya quedó.